### ISSUE TITLE
Prezentare mai îngustă a conjugărilor de verbe

### ISSUE THREAD

**Issue by CatalinFrancu** [dexonline/dexonline#125]
http://dexonline.ro/search.php?cuv=vrea

Din cauza reclamei, tabela cu flexiuni e împinsa în jos.
Cred că soluția ar fi să scriem variantele în tabelă unele sub altele

**Comment by CatalinFrancu** [comment]
Redenumesc acest tichet. Înțeleg că vrem o prezentare mai compactă a tabelelor pentru verbe, care să arate bine și pe dispozitive mobile.

**Comment by raduborza** [comment]
Da, exact!

**Comment by sandriu** [comment]
Momentan, pe mobil trebuie să scrollezi orizontal doar în portrait mode, ca să vezi tabelul în întregime.
În landscape mode se vede întreg:

![2016_03_20_21 01 04](https://cloud.githubusercontent.com/assets/1265882/13906545/5b8b25a0-eee1-11e5-8b67-f7c3eb027d5f.png)

Dacă inversăm tabelul, adică să afișăm pe coloane numărul și persoana, iar pe rânduri să punem timpii, atunci cel mai probabil că pe mobil va arăta bine (fără scroll orizontal în portrait mode), dar pe desktop se va mări necesitatea scroll-ului vertical (tabelul va fi mai îngust, dar mult mai lung). 
Dacă vrem să mergem pe principiul "Mobile first", atunci aceasta este soluția (să inversăm tabelul).
Dacă rămânem la paradigma clasică ("desktop first"), atunci tabelul ar trebui să rămână ca atare.

Am căutat "best practices" pe [Google Material Design](https://www.google.com/design/spec/material-design/introduction.html), unde scrie puțin despre responsive design aplicat pe tabele, dar în principiu ei zic să lași scroll orizontal, pentru tabelele foarte late.
Am mai căutat și pe [UI Patterns](http://ui-patterns.com/patterns), dar acolo n-am găsit nimic legat de tabele. 
**Aș mai face un pic de research înainte de a alege o soluție.**

Oricum, partea de responsive design la tabele cam intră în tichetul cu responsive design aplicat întregului site.

**Comment by CatalinFrancu** [comment]
Mi se pare rezonabil să ocupe mai mult loc pe verticală (deci să inversăm tabelul). Cine apasă linkul de expandare se așteaptă la asta.

E ok și să nu facem nimic, nu e chiar atât de supărător. Iar conjugarea pentru „a vrea” este și excepțional de lată, căci sunt multe variante (4 la m.m.c.perf.). La alte verbe tabelul este mai îngust.

Poate profităm de ocazie să îmbunătățim prezentarea. Există două capete de tabel: _infinitiv, infinitiv lung, participiu, gerunziu, imperativ,_ respectiv _prezent_ și celelalte moduri/timpuri personale. Reprezentarea aceasta e compactă, se încadrează frumos într-un dreptunghi, dar de câte ori am căutat concret ceva în tabel mi-a luat mult să găsesc.

**Comment by raduborza** [comment]
Ca alternativă ar fi și să spargem tablul în mai multe și să le afișăm unul
sub altul.
Cred că variantele ar trebui puse una sub alta, eventual să fie afișate
doar la cerere (să avem un buton/o bifă care să facă asta).

Radu

În data de 21 martie 2016, 12:59, Cătălin Frâncu <notifications@github.com>
a scris:

**Comment by sandriu** [comment]
Eu nu aș recomanda să afișăm la cerere variantele de conjugări, fiindcă asta ar însemna un click în plus pentru utilizator; ori utilizatorul și-a arătat deja intenția de a vedea conjugările în momentul în care a dat click pe link-ul "conjugări". În plus, afișarea la cerere a unui timp anume presupune că utlizatorul știe care este timpul pe care-l caută, dar dacă căutarea lui se face în sens invers, adică el are o formă de conjugare și vrea să afle ce timp are acea conjugare? 
Cred că afișarea integrală servește cel mai bine procesului de învățare a conjugărilor de verbe.

**Comment by sandriu** [comment]
Am tot căutat exemple de tabele de conjugări în română și în franceză (exemple: [1](http://conjugareaverbelor.ro/in-romana/conjugarea-verbului-vrea.html), [2](http://ro.bab.la/conjugare/franceza/savoir), [3](http://leconjugueur.lefigaro.fr/conjugaison/verbe/savoir.html), [4](http://www.les-verbes.com/conjuguer.php?verbe=savoir&submit.x=0&submit.y=0), [5](http://www.conjugaison.com/verbe/savoir.html), etc), dar niciunul nu mi-a plăcut.

Am încercat să construiesc propriul model de tabel pentru conjugare, împărțind informațiile existente pe moduri, timpuri, număr și persoană, dar am constatat că numărul și persoana (ca label-uri) reprezintă informație redundantă, care îngreunează într-un fel citirea tabelului:
![model-greoi-de-conjugare](https://cloud.githubusercontent.com/assets/1265882/14412107/4d15ca38-ff63-11e5-8515-db9859a0eab8.png)

**După care mi-a picat fisa**: să ne inspirăm de la oameni care fac _doar conjugări_: [Bescherelle: La Conjugaison Pour Tous](http://www.amazon.com/Bescherelle-Conjugaison-Francais-French-Edition/dp/2218951983/ref=dp_ob_title_bk) este o carte pentru conjugările verbelor în franceză. Modelul lor de conjugare pe moduri, timpuri și persoane mi se pare muuult mai lizibil și mult mai rapid de citit. Iată un [exemplu online](http://bescherelle.com/conjugueur.php?term=savoir), mai jos avem și screenshot-uri din ultima ediție:

![amazon com bescherelle la conjugaison pour tous - legenda tabel](https://cloud.githubusercontent.com/assets/1265882/14412149/81774058-ff64-11e5-9ea0-aa645ce0475c.png)
![amazon com bescherelle la conjugaison pour tous - exemplu de tabel explicat](https://cloud.githubusercontent.com/assets/1265882/14412153/a1a6a634-ff64-11e5-84a8-c29b4ca60e19.png)
![amazon com bescherelle la conjugaison pour tous - exemplu de verb conjugat](https://cloud.githubusercontent.com/assets/1265882/14412155/ab5436c4-ff64-11e5-9180-cef9a763a6b2.png)

Ce ziceți să structurăm tabelul de conjugare similar celui din Bescherelle?

**Comment by raduborza** [comment]
Utilizatorul și-a manifestat intenția de a vedea conjugările *corecte* cînd
a dat click. Dacă vrea să le vadă pe cele incorecte (eveniment rar), cineva
poate da încă un click. Altfel întotdeauna vor exista persoane care vor
zice: e în listă, deci e corect! Nu cred că vrem astea (chiar dacă există
culori diferite pentru variante)!

În ceea ce privește tabelele, înainte de a ne hotărî asupra unei variante
aș vrea să văd și cum ar arăta și tabelele cu declinările (la substantive
și adjective mai ales).

Radu

**Comment by CatalinFrancu** [comment]
Ei, hai că lumea se prinde că-s greșite dacă le scrii cu roșu, e o convenție destul de încetățenită. Sau cu galben (avertisment) pe cele nerecomandate. Dar mă rog, nu e așa important.

Îmi place stilul lui Bescherelle, să zicem modelul cu _protéger_. Cred că putem ascunde numărul și persoana și pune în loc _eu, tu, el/ea..._. Și arată mult mai citeț cu fiecare timp separat. Ca încadrare, putem pune două-trei coloane, iar când trecem la Bootstrap le facem să se înfășoare de la sine.

Putem rezolva separat substantivele / adjectivele, n-aș ține în loc implementarea pentru asta. Acolo sunt mult mai puține forme, dar nu mai face artificiul cu _eu, tu, el/ea_, deci s-ar putea să rămânem cu capete de tabel mari (_masculin, articulat, dativ-genitiv..._).

**Comment by raduborza** [comment]
Dacă punem o mică explicație în widgetul ăla (la început!), merge și cu
codul culorilor.
Măcar că majoritatea celor care ne folosesc sînt elevi care habar nu au pe
ce lume trăiesc...

Eventual nu punem legenda aia dacă nu e cazul (dacă nu avem roșu/galben).

R.

**Comment by CatalinFrancu** [comment]
De acord cu explicația. Facem asta deja, și pare că afișăm legenda doar când e cazul. Atâta că o afișăm sub paradigmă și cu albastru.

**Comment by CatalinFrancu** [comment]
FYI, cred că mingea aici e în terenul Soniei. Radu zicea că ar vrea să vadă și tabele pentru substantive și adjective, tot în stilul lui Bescherelle. Ei nu se ocupă de asta, deci pe site-ul lor nu vom găsi. Dar sper că nu-s greu de făcut, că sunt mai puține forme decât la verbe.

**Comment by sandriu** [comment]
Momentan nu mă mai ocup de dexonline (cel puțin în următoarele 4 luni), așa că mă voi retrage de pe acest tichet.
